arm-hdinsight/2015-03-01-preview/swagger/cluster.json

```diff
@@ -404,6 +404,9 @@
                 },
                 "configurations": {
                     "type": "object",
+                    "additionalProperties": {

```

Comment:
This's still breaking changes. In .Net, it became a Dictionary instead of an object.